src/i18n/ja/index.js

```diff
@@ -1,11 +1,156 @@
 import potato from './potato';
+import print from './print';
+import tester from './tester';
 export default {
   ja: {
     message: {
       ...potato,
+      help: {
+        label: 'ウィザード召喚'
+      },
+      print: { ...print },
+      tester: { ...tester },
       keyboard: {
-        label: 'きーぼーど'
-      }
+        label: 'keyboard'
+      },
+      layout: {
+        label: 'layout'
+      },
+      keymapName: {
+        label: 'Keymap Name',
+        placeholder: 'カスタムキーマップ名'
+      },
+      loadDefault: {
+        label: 'Load Default',
+        title: 'QMK Firmwareからデフォルトキーマップをロード'
+      },
+      compile: {
+        label: 'Compile',
+        title: 'キーマップのコンパイル'
+      },
+      downloadKeymap: {
+        title: 'keymap.cのみダウンロード',
+        label: 'keymap only'
+      },
+      downloadSource: {
+        title: 'ソースコードのダウンロード',
+        label: 'Full Source'
+      },
+      downloadJSON: {
+        title: 'QMK Keymap JSONファイルをエクスポート',
+        label: 'Keymap.JSON'
+      },
+      importJSON: {
+        title: 'QMK Keymap JSONファイルをインポート'
+      },
+      importUrlJSON: {
+        title: 'URLからQMK Keymap JSONをインポート'
+      },
+      printKeymap: {
+        title: 'キーマップレイヤの印刷',
+        label: 'Print Keymap'
+      },
+      testKeys: {
+        title: 'キーボード入力テスト',
+        label: 'Test Keyboard'
+      },
+      downloadFirmware: {
+        label: 'Firmware',
+        title: '書き込み用ファームウェアファイルのダウンロード'
+      },
+      flashFirmware: {
+        label: 'Auto-Flash',
+        title: 'コンパイルされたファームウェアをMCUに自動的に書き込む'
+      },
+      flashFile: {
+        label: 'Custom-Flash',
+        title: 'ユーザが選択したファイルをMCUに書き込む'
+      },
+      ColorwayTip: {
+        title: 'Ctrl + Alt + Nで次の配色に切り替え'
+      },
+      layer: {
+        label: 'Layer',
+        confirm: 'レイヤをクリアしますか？',
+        title: 'レイヤのクリア'
+      },
+      keymap: {
+        label: 'Keymap'
+      },
+      downloadToolbox: {
+        label: 'QMK Toolboxを入手'
+      },
+      keycodes: {
+        label: 'Keycodes'
+      },
+      keycodesRef: {
+        label: 'キーコードの参照'
```

Comment:
```suggestion
        label: 'キーコードの説明'
```
or
```suggestion
        label: 'キーコードの説明はこちら'
```
To make the meaning clearer.
ちょっと原文から離れる気もしますが、こちらのほうが分かりやすくないでしょうか？

Reply:
Resolved in a6944e4